src/utils/join.js

```diff
@@ -1,7 +1,11 @@
-// For some reason path.posix.join is undefined in webpack
-// Also, this is just much smaller
-import { normalizePath } from './normalizePath.js'
+const { normalize } = require('path').posix || require('path')
```

Comment:
This import statement is not correct as it is not compatible with the isomorphic part of isomorphic-git and its builder. Instead, it should be:

```
import { normalize } from './path.js'
```

Note that the path.posix abstraction is already provided by isomorphic-git.

Reply:
A `normalize` export could also be added to path.js so that it's possible to just import the normalize function.